src/tracer.js

```diff
@@ -131,12 +136,34 @@ const performPromisifyType = (err, data, type, callback) => {
   }
 };
 
+export const performStepFunctionLogic = handlerReturnValue => {
+  return (
+    safeExecute(() => {
+      if (!isStepFunction()) {
+        return handlerReturnValue;
+      }
+      const messageId = getRandomId();
+
+      addStepFunctionEvent(messageId);
+
+      const { data } = handlerReturnValue;
```

Comment:
how we sure it have data ?

Reply:
this is the format that `promisifyUserHandler` returns 🤔

Reply:
I'll destruct it in the beginning, and it will be clearer :)